labelbox/schema/project.py

```diff
@@ -312,18 +316,66 @@ def setup(self, labeling_frontend, labeling_frontend_options):
         timestamp = datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
         self.update(setup_complete=timestamp)
 
+    def validate_labeling_parameter_overrides(self, data):
+        for idx, row in enumerate(data):
+            if len(row) != 3:
+                raise TypeError(
+                    f"Data must be a list of tuples containing a DataRow, priority (int), num_labels (int). Found {len(row)} items. Index: {idx}"
+                )
+            data_row, priority, num_labels = row
+            if not isinstance(data_row, DataRow):
+                raise TypeError(
+                    f"Datarow should be be of type DataRow. Found {type(data_row)}. Index: {idx}"
```

Comment:
`Datarow should be be of type DataRow` kind of reads funny to me. like a  `cat should be a cat` feels kind of tautological

Reply:
updated it to the var name. data_row should be of type DataRow